src/main/java/org/craftercms/studio/impl/v1/deployment/PreviewDeployerImpl.java

```diff
@@ -85,7 +85,7 @@ public void subscribeToPreviewSyncEvents() {
 
     @EventListener(EVENT_PREVIEW_SYNC)
     public void onPreviewSync(PreviewEventContext context) {
-        doDeployment(context.getSite(), ENV_AUTHORING, context.isWaitTillDeploymentIsDone());
```

Comment:
Shouldn't we remove this property from the context too?

Reply:
The events for the preview target some times set it to `false`, not sure what the explanation is for each use case: db sync & cluster sync

Reply:
Ok, I thought the flag wasn't used somewhere else, my bad.

Reply:
I say we keep the option to wait.